text/0022-versions.md

```diff
@@ -0,0 +1,60 @@
+# Contributing guidelines
+- Start Date: 2020/02/06
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/22
+
+## Decision, including impact on distributions
+OpenMRS frontend ESMs should be versioned according to
+[Semantic Versioning](https://semver.org/), with the following modification
+to the definition of a major version:
+
+A **Major** version is one in which **any** of the following have taken place
+- Backward-incompatible changes have been made to the API, if the ESM exposes an API
+- Backward-incompatible changes have been made to the Configuration API,
+  such that upgrading might cause a previously valid configuration to be invalid
+- A functional and substantive UI change has been introduced, which is by
+  default different from the previous UI
```

Comment:
I think this would be better phrased as "A functional and substantive UI change has been introduced such that the UI differs from the previous UI using the default settings"

Reply:
Yes, that is better, thank you!

Reply:
Sorry, I changed my mind and decided it wasn't better. Though you're right about the grammatical problem in the referent of "which." I changed it a bit.